airbyte-integrations/connectors/source-surveymonkey/source_surveymonkey/spec.json

```diff
@@ -4,24 +4,27 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "SurveyMonkey Spec",
     "type": "object",
-    "required": ["start_date"],
+    "required": ["access_token", "start_date"],
```

Comment:
This will introduce a breaking change 🚨 ... what's the logic behind this now becoming mandatory?

Reply:
access_token - is required for OSS and it is also part of cred config after Oauth